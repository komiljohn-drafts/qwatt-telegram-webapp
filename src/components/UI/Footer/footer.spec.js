import React from 'react'
import { render } from '@testing-library/react'
import Footer from './index'

describe('About component', () => {
  it('should render Footer component', () => {
    const { asFragment } = render(<Footer />)
    expect(asFragment(<Footer />)).toMatchSnapshot()
  })
})
