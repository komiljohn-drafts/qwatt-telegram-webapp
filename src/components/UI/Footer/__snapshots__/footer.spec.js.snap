// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render Footer component 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="box"
      >
        <a
          class="logo"
          href="/"
        >
          <h2>
            Logo
          </h2>
        </a>
        <nav>
          <ul>
            <li>
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="/blog"
              >
                Blog
              </a>
            </li>
            <li>
              <a
                href="/about"
              >
                About
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
